CruGlobal/staff-conf-onsite
# Adding new fields to import script for child and student medical history

To test you will need to have rails running , reddis running and sidekiq running.

The hybrid import data  I am running is the following csv content  located here:
https://gist.github.com/kharperballistiq/280c26b9240f5edffdb3d073d302c885#file-cru-merged-sample-csv

To test a working example navigate to :
http://localhost:3000/families
![image](https://user-images.githubusercontent.com/26168983/51139377-14482700-181a-11e9-83ac-78b560e52790.png)
-=-=-=-=-=-=-=-=-=-=-=-=


Click on "**Import Spreadsheet**" in the top right to access the import screen
![image](https://user-images.githubusercontent.com/26168983/51139444-51141e00-181a-11e9-98bb-2d884f1caebc.png)
-=-=-=-=-=-=-=-=-=-=-=-=

Then choose the sample CSV file listed above and click submit.

After the import is complete, you can click on People>Families take you back to the people screen where you can see the name of a family listed.

If you want to check the database directly you should see the values imported into the **cru_student_medical_histories** and the **childcare_medical_histories tables**
